### DISCUSSION TITLE
What is the longest English word without repeated letters?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1026] [Galaxy Trivia] [answered]
**Bonus:** What is the longest single-vowel word in English?

**Comment by peterbaccaro** [comment, marked as answer]
The longest word in the English language without repeating any letter is **subdermatoglyphic**, a medical term, containing 17 letters.

**Strengths** is the longest single-vowel word in the English language.

**Reply by ParanoidUser** [reply]
I was thinking of *uncopyrightable*, which is 15 letters. But you broke the record once again! 👏👏

<img src="https://github.com/ParanoidUser/yolo/assets/5120290/ac0dea1e-76f1-48d2-9081-8dd3e58dd3aa" alt="record breaker" width="200"/>
